src/components/m-config-vue3.vue

```diff
@@ -20,37 +20,50 @@ export default defineComponent({
   components: {
     Vue3JsonEditor, 
   },
-  setup () {
+  props: ['session'], 
+  setup (props) {
     function onJsonChange (value) {
       console.log('value:', value)
+      state.json = value; 
+    }
+    function currentConfig(){
+      return state.json; 
+    }
+ 
+    var state; 
+    if (props.session === null){
+      state = reactive({
+        json: {'batch_size': 64,
+                    'logit_scale_init': 3.7,
+                    'image_loss_weight':1.,
+                    'device':'cuda:0',
+                    'opt_config': {'logit_scale': null, 
+                      'transformer': null,
+                      'transformer.resblocks.0.ln_': {'lr': 0.0007435612322566577,'weight_decay': 1.5959136512232553e-05},
+                      'transformer.resblocks.11.ln': {'lr': 0.0001298217305130271,'weight_decay': 0.015548602355938877},
+                      'ln_final': {'lr': 0.007707377565843718,'weight_decay': 0.0},
+                      'text_projection': {'lr': 5.581683501371101e-05, 'weight_decay': 0.0},
+                      'positional_embedding':null,
+                      'token_embedding':null,
+                      'visual': null,
+                      'positiional_embedding':null},
+                    'num_warmup_steps': 5,
+                    'rounds': 4,
+                    'margin':.3,
+                    'num_workers': 20,
+                    'test_size': 1000,
+                    'val_batch_size': 500}
+      });
+    } else { 
+      state = reactive({
+        json: props.session.params.method_config, 
```

Comment:
I'd like for the whole props.session.params to be editable, that would allow me to change more params.